blob/azureblob/azureblob.go

```diff
@@ -96,6 +96,8 @@ type Options struct {
 	// delegated privileges.
 	// See https://docs.microsoft.com/en-us/azure/storage/common/storage-dotnet-shared-access-signature-part-1#shared-access-signature-parameters.
 	SASToken SASToken
+
+	CloudEnvironment CloudEnvironment
```

Comment:
Can you add a comment here explaining what this option is, what the possible values are, and what it defaults to?